gsa_admin.py

```diff
@@ -1256,6 +1294,21 @@ def runCusSscript(self, sscript_file, out_fd, timeout=180):
     gsaWI.exportKeymatches(options.frontend, f)
     f.close()
 
+  elif action == "recrawl_pattern":
+    if not options.urlpattern:
```

Comment:
please reverse the order of these checks to avoid negation (which is less readable IMHO)

```
if options.urlpattern:
  pattern = options.urlpattern
else:
  # log error and exit
```